$Id$
excluding nextafter, does not build (strange macro is missing)
--- ltp-full-20061121.orig/testcases/misc/math/Makefile	2006-11-21 21:31:27.000000000 +0100
+++ ltp-full-20061121/testcases/misc/math/Makefile	2006-12-06 11:54:56.000000000 +0100
@@ -1,4 +1,4 @@
-SUBDIRS = abs atof float fptests nextafter
+SUBDIRS = abs atof float fptests
 
 all:
 	@set -e; for i in $(SUBDIRS); do $(MAKE) -C $$i ; done
